src/components/Comments/Comment.js

```diff
@@ -13,7 +13,7 @@ import MemoIc_arrow_upvote_off from '../../assets/arrow/Ic_upvote_off';
 import MemoIc_downvote_on from '../../assets/arrow/Ic_downvote_on';
 import MemoIc_upvote_on from '../../assets/arrow/Ic_upvote_on';
 import useComment from './hooks/useComment';
-import {FONTS} from '../../utils/theme';
+import {COLORS, FONTS} from '../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been updated to include `COLORS` from the theme file. This is a good change as it allows for more consistent color usage across the application.
```diff
- import {FONTS} from '../../utils/theme';
+ import {COLORS, FONTS} from '../../utils/theme';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->